lib/bb/configuration.rb

```diff
@@ -38,6 +38,8 @@ def connection
         # conn.request(:curl, ::Logger.new(STDOUT), :warn) unless Rails.env.production?
         # conn.response(:logger, ::Logger.new(STDOUT), bodies: true) unless Rails.env.production?
 
+        conn.response :betamocks if Settings.mhv.bb.mock
+
```

Comment:
Small nit.  Can you pls remove this space so that all the `conn.response`s are grouped together.